.github/workflows/ci.yml

```diff
@@ -15,74 +15,57 @@ on:
     tags: [v*]
 
 env:
-  PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
```

Comment:
I guess it was never used?

Reply:
It was moved to the step envs instead of the global one